src/network/backbox/rest/mode/intellicheck.pm

```diff
@@ -0,0 +1,152 @@
+#
+# Copyright 2024 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package network::backbox::rest::mode::intellicheck;
+
+use base qw(centreon::plugins::templates::counter);
+
+use strict;
+use warnings;
+use centreon::plugins::templates::catalog_functions qw(catalog_status_threshold_ng);
+
+sub prefix_intellicheck_output {
+    my ($self, %options) = @_;
+
+    return "Intellicheck " . $options{instance_value}->{id} . " '" . $options{instance_value}->{name} . "' - ";
+}
+
+sub set_counters {
+    my ($self, %options) = @_;
+
+    $self->{maps_counters_type} = [
+        { name => 'intellichecks', type => 1, cb_prefix_output => 'prefix_intellicheck_output', message_multiple => 'All intellichecks are ok', skipped_code => { -10 => 1 } },
+    ];
+
+    $self->{maps_counters}->{intellichecks} = [
+        { label => 'total', nlabel => 'intellicheck.total.count', set => {
+            key_values      => [ { name => 'total' } ],
+            output_template => 'total: %d',
+            perfdatas       => [
+                { value => 'total', template => '%d', min => 0, label_extra_instance => 1 }
+            ]
+        }
+        },
+        { label => 'success', nlabel => 'intellicheck.success.count', set => {
+            key_values      => [ { name => 'success' } ],
+            output_template => 'success: %d',
+            perfdatas       => [
+                { value => 'success', template => '%d', min => 0, label_extra_instance => 1 }
```

Comment:
We could add the `total` as `max` value for this counter (and the following ones).

Reply:
```suggestion
            key_values      => [ { name => 'success', name => 'total' } ],
            output_template => 'success: %d',
            perfdatas       => [
                { value => 'success', template => '%d', min => 0, max => 'total', label_extra_instance => 1 }
```